README.md

```diff
@@ -149,6 +149,9 @@ get a `numpy` array or `AICSImage.get_image_dask_data` to get a `dask` array for
 specific chunk of data from the image.
 * **If you are using a distributed cluster:** all functions and properties in the
 library are generally optimal.
+* **If you are using a distributed cluster but don't want image reads to be
+distributed:** use `aicsimageio.disable_dask()`. Useful when you want to use a
+distributed cluster but only have access to a few workers. Common on GPU workloads.
```

Comment:
What is it about small numbers of workers that makes this slower than non-distributed reads?  And what is it about GPU workloads that makes this common?

Reply:
Commonly, clusters have less GPU nodes than CPU nodes? :shrug: Will take suggestions on language here.

Reply:
My question as well - wouldn't this help at least a little bit?

Reply:
From all the testing I have done, on local machines, on the SLURM cluster, etc. you don't get too much performance improvement until about 8ish workers.

Reply:
Latest tests I've been running show quite the opposite of an improvement - ie. image reads taking 5-10 minutes with 3 workers versus 1-3 seconds when running outside of Dask.

Reply:
I think that agrees with my above statement?

> You don't get an improvement until 8ish workers

> 5 - 10 minutes with 3 workers. 1- 3 seconds when running outside of Dask

Reply:
I have updated the language to reflect my testing